journals/apps/core/models.py

```diff
@@ -31,3 +34,76 @@ def get_full_name(self):
     @python_2_unicode_compatible
     def __str__(self):
         return str(self.username)
+
+
+class SiteConfiguration(models.Model):
+    """
+    Each site/tenant should have an instance of this model. This model is responsible for
+    providing databased-backed configuration specific to each site.
+    """
+
+    site = models.OneToOneField('wagtailcore.Site', null=False, blank=False, on_delete=models.CASCADE)
+
+    lms_url_root = models.URLField(
+        verbose_name=_('LMS base url'),
+        help_text=_("Root URL of this site's LMS (e.g. https://courses.stage.edx.org)"),
+        null=False,
+        blank=False
+    )
+
+    lms_public_url_root_override = models.URLField(
+        verbose_name=_('LMS public base url'),
+        help_text=_(
+            "Root public URL of this site's LMS. Only used for overwriting client-side URLs"
+        ),
+        null=True,
+        blank=True
+    )
+
```

Comment:
Should we add discovery_url_root and ecommerce_url_root as well? I can do that in a separate PR if more appropriate

Reply:
I'd say we add them with a PR that needs them so they are well tested.